Foundation/NSURL.swift

```diff
@@ -309,7 +309,7 @@ open class NSURL : NSObject, NSSecureCoding, NSCopying {
 
         let thePath = _standardizedPath(path)
         
-        var isDir : Bool = false
+        var isDir: ObjCBool = false
```

Comment:
I don't understand why this is necessary... It's only going to get passed to an API which takes `bool` so can't it stay as `bool`?

Reply:
Because it calls 
`let _ = FileManager.default.fileExists(atPath: absolutePath, isDirectory: &isDir)`
and thats defined as:
`open func fileExists(atPath path: String, isDirectory: UnsafeMutablePointer<ObjCBool>?) -> Bool`

and `ObjCBool` is not a simple `Bool` anymore.

Reply:
Ohhhh, thanks.